Content.Server/Construction/AnchorableSystem.cs

```diff
@@ -85,7 +89,26 @@ private void OnAnchorComplete(EntityUid uid, AnchorableComponent component, TryA
             // TODO: Anchoring snaps rn anyway!
             if (component.Snap)
             {
-                _transform.SetCoordinates(uid, xform.Coordinates.SnapToGrid(EntityManager, _mapManager));
+                var coordinates = xform.Coordinates.SnapToGrid(EntityManager, _mapManager);
+
+                if (_tagSystem.HasTag(uid, "Unstackable"))
+                {
```

Comment:
Should use constant strings for stuff like this.

Reply:
Are you sure? Nothing else has.
![image](https://github.com/space-wizards/space-station-14/assets/5285589/9839d339-5e1c-4b7f-b2da-993493634741)